### PR TITLE
[UI/UX:TAGrading] Fix TA Grading Control Button Dark Mode

### LINKED ISSUE #10711: TA grading control buttons have ugly white outline in dark mode
**Describe the bug**
The control buttons in the TA grading interface have an ugly white outline in dark mode.

image

**Expected behavior**
The TA grading control buttons should have a clean design in both light and dark modes.

### PR DESCRIPTION
**Pull request by lukeckk**
### What is the current behavior?
Edit_2

Closes #10711 
### What is the new behavior?

In Dark mode:
image

In Dark/Black mode:
image

### CODE REVIEW

**Comment by lukeckk** [comment]
This solution is for issue #10711, not sure why it switched to #10723 by itself. Sorry for the confusion!

File: junit_test_runner/TestRunner.java [hunk 1]
```diff
@@ -13,6 +13,7 @@ public class TestRunner {
     private static String TESTS_SUFFIX = "Test";
 
     /*
+    doc test
```
Comment (williamschen23) [inline junit_test_runner/TestRunner.java:16]: remove

File: site/public/css/electronic.css [hunk 2]
```diff
@@ -175,9 +175,9 @@ button > i.fas:hover {
 .ta-navlink-cont {
     position: relative;
     margin-right: 4px;
-    border: 2px solid var(--default-black);
+    /*border: 2px solid var(--default-black);*/
```
Comment (williamschen23) [inline site/public/css/electronic.css:178]: There should be an outline. The main problem is that it does not look good on dark mode, but there should still be an outline

File: site/public/css/electronic.css [hunk 3]
```diff
@@ -175,9 +175,9 @@ button > i.fas:hover {
...
     border-radius: 5px;
-    box-shadow: 2px 2px 4px;
+    /*box-shadow: 2px 2px 4px;*/
```
Comment (williamschen23) [inline site/public/css/electronic.css:180]: same here

**Review by williamschen23 — CHANGES_REQUESTED** [review]

File: site/public/css/electronic.css [hunk 4]
```diff
@@ -171,13 +171,25 @@ button > i.fas:hover {
...
 
+:root {
+    --default-border-color: black;
+    --default-box-shadow: 2px 2px 4px black;
+}
+
+@media (prefers-color-scheme: dark) {
+    :root {
+        --default-border-color: grey;
+        --default-box-shadow: 2px 2px 4px grey;
```
Comment (micpap25) [inline site/public/css/electronic.css:182]: This color choice causes the outline to not be visible in the Dark (Black) mode.
image

**Review by micpap25 — DISMISSED** [review]

File: site/public/css/electronic.css (hunk 4 above)
Comment (lukeckk) [reply site/public/css/electronic.css:182]: Screenshot 2024-07-19 at 8 55 19 AM

It is working on my end in the Dark (Black) mode. Im currently testing it on the browser's inspection tool. How should I go about to perform more testing?

**Comment by williamschen23** [comment]
> Just pushed up new changes. Below is the way it will look in dark mode. :)
> 
> After

Dismissed Michael's comment because it works on my end.
Can you update the main PR message. I also personally do not like the heavy border shadow, can it be lowered? The final review should go to a maintainer, so cc @williamjallen

**Comment by lukeckk** [comment]
Screenshot 2024-07-19 at 10 55 57 AM

@williamschen23  This is how it will look after lowering the shadow horizontally and vertically by 1px and the blur radius by 2px. What do you think ?

**Comment by williamschen23** [comment]
I do not know much about changing CSS based on different themes, but looking at colors.css we use this syntax instead
image

**Comment by lukeckk** [comment]
Got it. But looking at the image i included in the previous comment with the reduced shadow. Are you ok with that? Wanted to double check before I go ahead and push the changes

**Comment by lukeckk** [comment]
Awesome! One more question regarding the title. Im aware that there is a certain way to write it so it will close the issue. Would it be something like below? 

Fixes for #10723

**Comment by williamschen23** [comment]
I added it, Closes #issue number should close it when completed.

**Comment by williamschen23** [comment]
Also don't forget to update the descriptioin of the PR for the new changes

**Comment by lukeckk** [comment]
Will do! I'll push it up in a few hours after work. Thank you!!

File: site/public/css/electronic.css [hunk 5]
```diff
@@ -171,13 +171,25 @@ button > i.fas:hover {
     color: initial;
 }
 
+:root {
+    --default-border-color: black;
+    --default-box-shadow: 1px 1px 2px black;
+}
+
+@media (prefers-color-scheme: dark) {
```
Comment (williamschen23) [inline site/public/css/electronic.css:179]: Use [data-black_mode="black"][data-theme="dark"] like in colors.css

**Review by williamschen23 — CHANGES_REQUESTED** [review]

File: site/public/css/electronic.css (hunk 5 above)
Comment (williamschen23) [reply site/public/css/electronic.css:179]: Michael cannot see the css if it does this, I do not know why.

Comment (williamschen23) [reply site/public/css/electronic.css:179]: Update you checked the OS's color scheme instead of Submitty light scheme, comment still stands.

Comment (lukeckk) [reply site/public/css/electronic.css]: got it! just pushed up the changes. Please check

**Comment by lukeckk** [comment]
Please ignore the commit titled "Edited error message for submission page for grading issues to display more information", that was meant for a different issue

**Comment by micpap25** [comment]
@lukeckk please revert this commit or force-push over it, thank you!

**Review by powe97 — APPROVED** [review]
Looks good

**Pull request merged**